src/Chainweb/Version.hs

```diff
@@ -658,12 +652,8 @@ pactBackCompat_v16 _ _ = False
 -- dependency of `newBlock` on the creation time of the current header can be
 -- removed.
 --
-useLegacyCreationTimeForTxValidation
-    :: ChainwebVersion
-    -> BlockHeight
-    -> Bool
+useLegacyCreationTimeForTxValidation :: ChainwebVersion -> BlockHeight -> Bool
 useLegacyCreationTimeForTxValidation Mainnet01 h = h < 449940 -- ~ 2020-04-03T00:00:00Z
-useLegacyCreationTimeForTxValidation Development h = h < 150
```

Comment:
Having this in devnet was annoying and slowing down testing. I don't thing we need to cover this in devnet any more.